docs/development.rst

```diff
@@ -14,17 +14,48 @@ I am the primary author of Hypothesis.
 5. Planning the general roadmap of the project
 6. Doing sponsored development on tasks that are too large or in depth for other people to take on
 
-So all new features must either be sponsored or implemented by someone else. That being said, I take a fairly active
-role in shepherding pull requests and helping people write a new feature (see `the
-contributing guidelines <https://github.com/HypothesisWorks/hypothesis-python/blob/master/CONTRIBUTING.rst>`_ for
-details and `this pull request
-<https://github.com/HypothesisWorks/hypothesis-python/pull/154>`_ for an example of how the process goes). This isn't
+So all new features must either be sponsored or implemented by someone else.
+That being said, I take a fairly active role in shepherding pull requests and
```

Comment:
Updating the docs to reflect that Hypothesis is a team project would be nice, and also a fair bit more work than just replacing pronouns.  I'll come back another time.